api/addon.go

```diff
@@ -35,7 +35,7 @@ func (h AddonHandler) AddRoutes(e *gin.Engine) {
 // Get godoc
 // @summary Get an addon by name.
 // @description Get an addon by name.
-// @tags get
```

Comment:
hint: godoc tags were changed so that the generated docs are grouped by endpoint rather than by method